### PR TITLE
[RFC] functional/legacy/lstlbr_utf8_spec conversion to neovim test

### PR DESCRIPTION
**Pull request by teto**
I've been fiddling with column related code tested by this vim test. The results are really hard to interpret hence I chose to convert it to neovim tests.
This is not ready yet but I thought I would let others know to prevent duplicate work.

### CODE REVIEW

File: test/functional/ui/listlbr_utf8_spec.lua [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+local Screen = require('test.functional.ui.screen')
+
+local request = helpers.request
+
+local feed, execute, curbuf = helpers.feed, helpers.execute, helpers.curbuf
+local insert = helpers.insert
+local session = helpers
+local source = helpers.source
+
+if session.pending_win32(pending) then return end
```
Comment (justinmk) [inline test/functional/ui/listlbr_utf8_spec.lua]: Should be able to remove this line, screen tests work on Windows/AppVeyor.

File: test/functional/legacy/listlbr_utf8_spec.lua [hunk 2]
```diff
@@ -1,209 +1,184 @@
...
-      eeeeee>--->-;>some text                 
-      Test 6: Screenattributes for comment
-       /*		 and some more */
-      ScreenAttributes for test6:
-      Attribut 0 and 1 and 3 and 5 are different!
-      Test 8: set linebreak with visual block mode and v_b_A and selection=exclusive and multibyte char
-      long line: foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar foobar TARGETÃx' at end
+  -- TODO update with new neovim system to check attributes
+  -- or remove it if done somewhere else
+  -- it("screenattributes for comment", function()
```
Comment (teto) [inline test/functional/legacy/listlbr_utf8_spec.lua]: seems like a bad test compared to screen:expect . do you still want me to upgrade it ? or simply remove it ?

Comment (chrisbra) [inline test/functional/legacy/listlbr_utf8_spec.lua]: If i am not completly mistaken that is part of the new style Vim test test_listlbr_utf8.vim. I thought those are not converted?

Comment (jamessan) [inline test/functional/legacy/listlbr_utf8_spec.lua]: That test was only converted to a new style test in [8.0.0311](https://github.com/vim/vim/releases/tag/v8.0.0311).  The old style test was converted to a lua test over a year ago, and we haven't caught up to the new style test yet.

**Comment by justinmk** [comment]
@teto was this overruled by other work?

**Comment by teto** [comment]
Not that I know of, I was just indecisive about "screenattributes for comment" and then had not the time. Anyone feel free to pick it up.

**Comment by justinmk** [comment]
We merged `test_listlbr_utf8.vim`. Let's move on.

**Pull request closed without merging**